staging/cse/windows/debug/collectlogs.ps1

```diff
@@ -69,6 +69,9 @@ else
     @{Name="CompartmentID"; Expression={$_.NamespaceId}}, `
     @{Name="CompartmentGuid"; Expression={$_.NamespaceGuid}}, `
     Type, @{Name="EndpointID"; Expression={$_.Resources.Data.Id}} > namespaces.txt
+
+    # V2 HNS port pool allocations
+    (Invoke-HnsRequest -Type portpools -Method GET).PortPoolAllocations | ConvertTo-Json -Depth 10 > hnsportpoolallocs.txt

```

Comment:
Hi @princepereira . This script is originated from https://github.com/microsoft/SDN/blob/master/Kubernetes/windows/debug/collectlogs.ps1

You should update the origin repo and then leverage https://github.com/Azure/AgentBaker/blob/master/staging/cse/windows/debug/update-scripts.ps1
to synchronize it.

Reply:
For example, 
- https://github.com/microsoft/SDN/pull/602
- https://github.com/Azure/AgentBaker/pull/4279/files

Reply:
Thanks @ShiqianTao . Making the recommended changes now.

Reply:
Or you can modify [collect-windows-logs.ps1](https://github.com/Azure/AgentBaker/blob/master/staging/cse/windows/debug/collect-windows-logs.ps1) which managed by AgentBaker